treeherder/webapp/api/performance_serializers.py

```diff
@@ -17,8 +17,8 @@ class Meta:
 
 class TestOptionsSerializer(serializers.CharField):
     def to_representation(self, obj):
-        # if extra_propeties str is blank, just return nothing
-        if type(obj) == str:
+        # if extra_options str is blank, just return nothing
+        if type(obj) == unicode:
```

Comment:
I think the right thing to do here is to use the six library and see if it's a string type (this will be forewards compatible with python3 and doesn't tie the code to an implementation detail of django)
```py
import six
...
if isinstance(obj, six.string_types):
```
http://stackoverflow.com/a/20612311